mne/utils/docs.py

```diff
@@ -308,9 +308,10 @@ def _reflow_param_docstring(docstring, has_first_line=True, width=75):
 
 docdict["aseg"] = """
 aseg : str
-    The anatomical segmentation file. Default ``aparc+aseg``. This may
-    be any anatomical segmentation file in the mri subdirectory of the
-    Freesurfer subject directory.
+    The anatomical segmentation file. Default ``auto`` uses ``wmparc``
+    if available and ``aparc+aseg`` if not. This may be any anatomical
```

Comment:
```suggestion
    The anatomical segmentation file. Default ``auto`` uses ``aparc+aseg``
    if available and ``wmparc`` if not. This may be any anatomical
```